README.md

```diff
@@ -1,18 +1,18 @@
-# TU Delft CS4110 Software Testing and Reverse Engineering
+# TU Delft CS4110: Software Testing and Reverse Engineering
 
-2015-2016 Q3 -- [TU Delft Study Guide](http://studiegids.tudelft.nl/a101_displayCourse.do?course_id=37845)
+2015-2016 Q3 — [TU Delft Study Guide](http://studiegids.tudelft.nl/a101_displayCourse.do?course_id=37845)
 
 ## Collaboration
 
 ### Communication
 
-We are using Slack! Use [cs4110.slack.com](https://cs4110.slack.com/) and sign-up with your tudelft.nl or utwente.nl email address.
+We are using Slack! Use [cs4110.slack.com](https://cs4110.slack.com/) and sign-up with your TU Delft or UTwente email address.
 
 ### GitHub
 
 GitHub is used for collaboration on assignments. Teams (project groups) can be formed through the Classroom for GitHub interface; please find the invitation link in the Blackboard announcement.
 
-An empty repository is created for each team and we request you use that for work on the assignments. **Please create at least a README.md file with team member names and the topic of choice.** If you have any questions, ask on Slack.
+An empty repository is created for each team and we request you use that for work on the assignments. **Please create at least a README.md file with the team member names and the topic of choice.** If you have any questions, ask on Slack. **Please note these repositories are public.**
```

Comment:
They weren't in my previous course that used GitHub, IN4303 (compiler construction), so I think it's worth pointing out.